fastlane/docs/Actions.md

```diff
@@ -2438,6 +2438,12 @@ Install an Xcode plugin for the current user
 install_xcode_plugin(url: 'https://github.com/contentful/ContentfulXcodePlugin/releases/download/0.5/ContentfulPlugin.xcplugin.zip')
```

Comment:
Makes sense, do you think it's OK to use a fake URL? Don't actually know of any plugins not hosted on :octocat: